### DISCUSSION TITLE
Is it possible to use a link to load an image?

### DISCUSSION THREAD

**Discussion by TECHNOPAWS** [Brooooooklyn/canvas#474] [Q&A]
I'm probably blind but I have searched wherever I could and I finally had to come here, I used to use the [canvas package](https://www.npmjs.com/package/canvas) and in that I could just use `loadImage()`. I had to switch due to some reasons and I was just wondering if there was any similar way to `loadImage()` in @napi-rs/canvas

**Comment by twlite** [comment]
I have made a PR for `loadImage` function -> https://github.com/Brooooooklyn/canvas/pull/483
